run setup depends on chalk

Cloned repo branch v3.0.0
npm run setup

```
Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/jacob/test_projects/react-boilerplate/internals/scripts/helpers/checkmark.js:2:13)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

solved by manually running 
`npm install chalk` 

Should it really depend on the user having chalk installed before initializing this awesome project? It seems a bit silly for coloring a checkmark in an init script to have that requirement imho.

--- a/package.json
+++ b/package.json
@@ -17,6 +17,7 @@
     "start:tunnel": "cross-env NODE_ENV=development node server",
     "start:prod": "cross-env NODE_ENV=production node server",
     "pagespeed": "node ./internals/scripts/pagespeed.js",
+    "presetup": "npm i chalk",
     "setup": "node ./internals/scripts/setup.js",
     "clean": "shjs ./internals/scripts/clean.js",
     "generate": "plop --plopfile internals/generators/index.js",

